ui/src/app/utils/user-apps-utils.tsx

```diff
@@ -89,26 +90,19 @@ const appJustTurnedRunningFromProvisioning = (listAppsResponse) => {
         return !!runningAppFromApi && runningAppFromApi.length > 0;
       }
       return false;
-    });
-  return appsJustStartedRunning;
+    }) || []
+  );
 };
 
-const callLocalizeIfApplicable = (listAppsResponse) => {
+const callLocalizeIfApplicable = (listAppsResponse: ListAppsResponse): void => {
   // If userAppsStore is not updated lets wait for it to be updated before checking
-  if (!!userAppsStore.get() && userAppsStore.get().userApps === undefined) {
```

Comment:
don't need this first check.  we also want to quit if `userAppsStore.get()` is undefined.